lib/simplecontributionscheme.js

```diff
@@ -31,20 +31,86 @@ class SimpleContributionScheme extends ExtendTruffleContract(SoliditySimpleContr
     return new this(contract);
   }
 
-  async submitContribution(opts={}) {
+  async proposeContribution(opts={}) {
```

Comment:
why proposeContribution? the contract calls it submitContribution

Reply:
I am aiming for consistency in function naming.  As a result there are more than one whose names differ from the primary underlying contract function.  Further, when a proposal is involved, I want the function name to be explicit that we are proposing something, and again to be consistent in that nomenclature.  I don't see a lot of advantage in matching a contract function name.